price-feeder/oracle/provider/huobi.go

```diff
@@ -305,6 +302,14 @@ func (p *HuobiProvider) messageReceived(messageType int, bz []byte, reconnectTic
 			"provider",
 			config.ProviderHuobi,
 		)
+		return
+	}
+
+	if tickerErr != nil || candleErr != nil {
+		p.logger.Error().
+			AnErr("ticker", tickerErr).
+			AnErr("candle", candleErr).
+			Msg("Error on receive message")
 	}
```

Comment:
This if statement will always be true (with both errors being non-nil) - except in the very rare case where _both_ unmarshals succeeded, but candleResp.Tick.Close == 0 and tickerResp.Tick.LastPrice == 0.

I think that edge case can be ignored, so consider removing the "if" around the log statement.

Reply:
The kind of problem is that the Huobi and also Binance send a message every time a new channel is subscribed, like: 
![image](https://user-images.githubusercontent.com/17556614/160135479-dbaaffc1-185b-4f61-8d0d-8739a8bb6cc0.png)

We could add an unmarshal for these types, but it seems an overkill instead of having the if statement 
What do you think? @toteki